app/components/RunPanel.vue

```diff
@@ -48,6 +66,26 @@ const props = defineProps<{
   align-self: start;
 }
 
+.panel-actions {
```

Comment:
This looks quite ugly:
![Screenshot 2025-07-02 at 15 22 33](https://github.com/user-attachments/assets/82aa93e7-49c7-4a29-bbe8-3f7923cf3754)